packages/odyssey-react-mui/src/MenuButton.tsx

```diff
@@ -65,10 +82,25 @@ export type MenuButtonProps = {
    * If the MenuButton is an overflow menu or standard menu.
    */
   isOverflow?: boolean;
+  /**
+   * The horizontal alignment of the menu.
+   */
+  menuAlignment?: "left" | "right";
+  /**
+   * Optional function to fire on button click
+   */
+  onClick?: ButtonProps["onClick"];
+  /**
+   * Optional function to fire on memu close
+   */
+  menuPaperRef?: MutableRefObject<HTMLDivElement>;
+  onClose?: MenuProps["onClose"];
+  preventCloseOnChildClick?: boolean;
```

Comment:
This should also be `is` or `has`. Something like `hasCloseButton` which currently defaults to `true`. Or `isClosable`; although, that's less-effective.